### PR TITLE
Man page updates for report cli

### LINKED ISSUE #608: Add man page entry for CLI qpc report summary/detail commands
# Opening a new issue
___

## Specify type:
- Enhancement

## Description:
Add man page entry for CLI qpc report summary/detail commands
___

### PR DESCRIPTION
**Pull request by kholdaway**
Closes #608

### CODE REVIEW

File: docs/source/man.rst [hunk 1]
```diff
@@ -388,6 +388,48 @@ The ``qpc scan cancel`` command cancels the execution of a scan. A canceled scan
...
 
+Reports
+--------
+
+Use the ``qpc report`` command to view the reports from a scan.  Reports can be viewed as JSON or a CSV.  There are two different types of report: detail and summary.
+
+
+View Detail Report
+~~~~~~~~~~~~~~~~~~~
+The ``qpc report detail`` command provides unprocessed facts gathered during a scan.  These are the raw output from network, VCenter, and Satellite scans.
```
Comment (chambridge) [inline docs/source/man.rst]: did we uppercase VCenter and Satellite elsewhere?  at minimum I think its vCenter.

File: docs/source/man.rst [hunk 2]
```diff
@@ -388,6 +388,48 @@ The ``qpc scan cancel`` command cancels the execution of a scan. A canceled scan
...
+
+**qpc report detail --id** *scan_identifier* **(--json|--csv)** **--output-file** *PATH*
+
+``--id=scan_identifier``
+  Required. Contains the scan identifier to display.
+
+``--json``
```
Comment (chambridge) [inline docs/source/man.rst]: Example of how we did this in credentials.
https://github.com/quipucords/quipucords/blob/master/docs/source/man.rst#creating-and-editing-credentials

File: docs/source/man.rst [hunk 3]
```diff
@@ -388,6 +388,48 @@ The ``qpc scan cancel`` command cancels the execution of a scan. A canceled scan
...
+
+``--csv``
+  Optional. Displays the results of the report as CSV.
+
+``--output-file=PATH``
+  Required. Path to a file location where the report data will be saved.
+
+View Summary Report
+~~~~~~~~~~~~~~~~~~~
+The ``qpc report summary`` command provides processed fingerprints gathered during a scan.  Processed fingerprints are the result after merging facts from various sources.  Deduplication is also performed when possible.  For example, a system with an identical mac address could be seen during a network scan and a Vcenter scan.  The raw facts would be merged to provide a fingerprint consisting of information from both network and Vcenter data.
```
Comment (chambridge) [inline docs/source/man.rst]: Again don't think Vcenter is right, also inconsistent with previous section.

File: docs/source/man.rst [hunk 4]
```diff
@@ -388,6 +388,48 @@ The ``qpc scan cancel`` command cancels the execution of a scan. A canceled scan
...
+
+**qpc report summary --id** *scan_identifier* **(--json|--csv)** **--output-file** *PATH*
+``--id=scan_identifier``
+  Required. Contains the report's scan identifier to display.
+
+``--json``
+  Optional. Displays the results of the report as JSON.
```
Comment (chambridge) [inline docs/source/man.rst]: same comment for mutual exclusion as above.

File: docs/source/man.rst (hunk 3 above)
Comment (kholdaway) [reply docs/source/man.rst]: same

File: docs/source/man.rst (hunk 1 above)
Comment (kholdaway) [reply docs/source/man.rst]: all lower now.

File: docs/source/man.rst (hunk 2 above)
Comment (kholdaway) [reply docs/source/man.rst]: Done.

**Pull request merged**